javascript/chronometer.js

```diff
@@ -1,34 +1,47 @@
 class Chronometer {
-  constructor() {
-    // ... your code goes here
-  }
-
+  constructor() {}
+  currentTime = 0;
+  intervalId = null;
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1;
+    }, 1000);
```

Comment:
Don't forget to do something with the callback function, you will need it for the second part of the exercise